doc/code/qml_spin.rst

```diff
@@ -0,0 +1,15 @@
+qml.spin
+=========
+
+Overview
+--------
+
+This module contains functions and classes for creating and manipulating spin Hamiltonian operators.
+
+.. currentmodule:: pennylane.spin
```

Comment:
I suggest following the doc for [qml.debugging](https://docs.pennylane.ai/en/stable/code/qml_debugging.html) and manually define the functions to make sure that the Lattice object does not appear for now:

:html:`<div class="summary-table">`

.. autosummary::
    :nosignatures:

    ~pennylane.spin.transverse_ising
    ~pennylane.spin.heisenberg
    ~pennylane.spin.fermi_hubbard


:html:`</div>`